doc/schemas/getinfo.schema.json

```diff
@@ -100,7 +99,7 @@
     },
     "address": {
       "type": "array",
-      "description": "The addresses we announce to the world",
+      "description": "The addresses we announce to the world (omitted when not listening)",
```

Comment:
This does not seem to be the case?  In fact, we have other fields which are always present (our_features, mainly).

```
	/* Add network info */
	json_array_start(response, "address");
	if (cmd->ld->listen) {
		/* These are the addresses we're announcing */
```

i.e. `address` is always there, it's just any empty array?  And `binding` doesn't appear, which is weird, but a side effect of this branch.